src/smartMirrorProject/timeWidget/templates/timeWidget/time.html

```diff
@@ -0,0 +1,15 @@
+{% if data %}
+
+<div class="time-widget">
+
+    <!-- Load static files -->
+    {% load static %}
```

Comment:
Boven buiten de IF statement?

Reply:
Gefixed, misschien ook kijken voor andere widgets om dit te doen later. 

Zorgt ervoor dat de CSS van de widget blijft als er geen informatie beschikbaar is.